gradle/java.gradle

```diff
@@ -11,7 +11,7 @@ compileJava {
 }
 
 task testJar(type: Jar, dependsOn: testClasses, group: "build") {
-    appendix = 'test'
+    classifier = 'test'
```

Comment:
@michalkurka ahh, yes! you are right! i forgot that i already exported JVM_CLASSPATH env variable. 🎉